cellprofiler/modules/blobdetection.py

```diff
@@ -0,0 +1,379 @@
+# coding=utf-8
+
+"""
+
+Find blobs in a grayscale (single-channel) image or volume. Blobs are bright on dark or dark on bright regions in an
```

Comment:
Are you saying that it’s limited to single-channel two-dimensional and single-channel multi-dimensional images? Is there a reason this cannot be extended to multichannel two-dimensional and multichannel three-dimensional images? Is it possible to do so by pre-processing with `rgb2gray`?